types/setting.go

```diff
@@ -91,6 +94,16 @@ var (
 		ReadOnly:    false,
 	}
 
+	SettingDefinitionBackupstorePollInterval = SettingDefinition{
+		DisplayName: "Backupstore Poll Interval",
+		Description: "The polling interval for Longhorn to update backstore.",
```

Comment:
Typo.

Change it to:
`In seconds. The interval to poll the backup store for updating volumes' Last Backup field.`